src/js/components/Chart/Chart.js

```diff
@@ -254,17 +267,33 @@ const Chart = React.forwardRef(
 
           const key = `p-${index}`;
           // Math.min/max are to handle negative values
-          const bottom =
-            value.length === 2
-              ? Math.min(Math.max(0, bounds[1][0]), value[1])
-              : Math.min(value[1], value[2]);
-          const top =
-            value.length === 2
-              ? Math.max(Math.min(0, bounds[1][1]), value[1])
-              : Math.max(value[1], value[2]);
-          const d =
-            `M ${valueToCoordinate(value[0], bottom).join(',')}` +
-            ` L ${valueToCoordinate(value[0], top).join(',')}`;
+          let d;
+          if (direction === 'vertical') {
+            const left =
+              value.length === 2
+                ? Math.min(Math.max(0, bounds[0][0]), value[1])
```

Comment:
It might be nice to refactor some of the `bounds[0][0]` type of values to variables with more semantic meaning. Would make it a bit easier to follow how these map to the visuals and direction of the chart.